### DISCUSSION TITLE
How to set value for field in behavior

### DISCUSSION THREAD

**Discussion by reach2rv** [serenity-is/Serenity#6315] [Help] [answered]
Hi @VictorTomaili  @volkanceylan 

How can I assign value to a field fetched using Target.Fields.FindField("FieldName") inside behavior.  or there any other method to assign value?

**Comment by VictorTomaili** [comment, marked as answer]
Check existing behaviors you are probably can find a solution for your case. And please don't mention us when it's not important.

https://github.com/serenity-is/Serenity/tree/master/src/Serenity.Net.Services/RequestHandlers/IntegratedFeatures
